install_LGBM.232.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/bash
```

Comment:
same comment here. If this script is just from local development, please remove it and just include the code in the pull request description